test/integration/pack/test_pack.py

```diff
@@ -1732,9 +1733,291 @@ def test_rocks_noncache_flag(cartridge_cmd, light_project, tmpdir, pack_format):
     rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
     assert rc == 0
     assert "Using cached path .rocks" not in output
-    assert not os.path.exists(get_rocks_cache_path())
+    assert len(os.listdir(get_rocks_cache_path())) == 0
 
     rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
     assert rc == 0
     assert "Using cached path .rocks" not in output
-    assert not os.path.exists(get_rocks_cache_path())
+    assert len(os.listdir(get_rocks_cache_path())) == 0
+
+
+@pytest.mark.parametrize('pack_format', ['tgz'])
+def test_no_cache_yml_file(cartridge_cmd, light_project, tmpdir, pack_format):
+    project = light_project
+    remove_project_file(project, "pack-cache-config.yml")
+
+    if os.path.exists(get_rocks_cache_path()):
+        shutil.rmtree(get_rocks_cache_path())
+
+    cmd = [
+        cartridge_cmd,
+        "pack", pack_format,
+        project.path
+    ]
+
+    if platform.system() == 'Darwin':
+        cmd.append('--use-docker')
+
+    rc, output = run_command_and_get_output(cmd, cwd=tmpdir)
+    assert rc == 0
+    assert "Using cached path" not in output
+    assert "Failed to process pack-cache-config.yml file which contain cache paths" not in output
```

Comment:
Make this check simpler: `assert "Failed to process pack-cache-config.yml file" not in output`

You don't need a whole message.